src/spork.cpp

```diff
@@ -84,7 +84,10 @@ bool CSporkManager::UpdateSpork(int nSporkID, int64_t nValue)
 {
     CSporkMessage spork = CSporkMessage(nSporkID, nValue, GetTime());
 
-    spork.Sign(strMasterPrivKey);
+    if (!spork.Sign(strMasterPrivKey)) {
+        LogPrintf("CSporkManager::UpdateSpork -- fail to generate valid signature\n");
```

Comment:
why not use `return error("CSporkManager::UpdateSpork -- fail to generate valid signature");` here?

Reply:
you're right. but I already remove the change. thanks anyway.